doc/src/documentation_style_guide.rst

```diff
@@ -0,0 +1,1019 @@
+The purpose of these docstrings are to explain the API of that class or
+function. The doctests examples are tested as part of the test suite, so that we
+know that they always produce the output that they say that they do. Here is an
+`example docstring
+<https://github.com/sympy/sympy/blob/b176f6a1d9890b42dc361857c887992315e3d5ad/sympy/functions/elementary/complexes.py#L22-L47>`_.
+Most docstrings are also automatically included in the Sphinx documentation
+above, so that they appear on the SymPy Documentation website. Here is that
+`same docstring
+<https://docs.sympy.org/latest/modules/functions/elementary.html#sympy.functions.elementary.complexes.im>`_
+on the SymPy website. The docstrings are formatted in a specific way so that
+Sphinx can render them correctly for the docs website. The SymPy sources all
+contain sparse technical documentation in the form of source code comments,
+although this does not generally constitute anything substantial and is not
+displayed on the documentation website.
+
+**SymPy Wiki** https://github.com/sympy/sympy/wiki
+
+The SymPy Wiki can be edited by anyone without review. It contains various
+types of documentation, including:
+
+* High-level developer documentation (for example: https://github.com/sympy/sympy/wiki/Args-Invariant)
+* Guides for new contributors (for example: https://github.com/sympy/sympy/wiki/Introduction-to-contributing)
+* Development policies (for example: https://github.com/sympy/sympy/wiki/Python-version-support-policy)
+* Release notes (for example: https://github.com/sympy/sympy/wiki/Release-Notes-for-1.5)
+* Various pages that different contributors have added
+
+Narrative Documentation Guidelines
+==================================
+
+Extensive documentation, or documentation that is not centered around an API
+reference, should be written as a narrative document in the Sphinx docs (located
+in the `doc/src directory
+<https://github.com/sympy/sympy/tree/master/doc/src>`_). The narrative documents
+do not reside in the Python source files, but as standalone restructured files
+in the doc/src directory. SymPy’s narrative documentation is defined as the
+collective documents, tutorials, and guides that teach users how to use SymPy.
+
+Steps to Writing Documentation
+------------------------------
+
+The first step to contributing to the code base is creating your development
+environment. Please find instructions on how to create your development
+environment in our `Development Workflow – Create Your Environment
+<https://github.com/sympy/sympy/wiki/Development-workflow#create-your-environment>`_
+guide.
+
+Once you have created your development environment, follow these steps:
+
+1. Installation
+---------------
+
+**Debian/Ubuntu**
+
+For Debian/Ubuntu, install the prerequisites::
+
+   apt-get install python-sphinx texlive-latex-recommended dvipng librsvg2-bin
+   imagemagick docbook2x graphviz python -m pip install sphinx-math-dollar
```

Comment:
I guess that the `python -m pip ...` is a separate comment that should be on a separate line. I'm not sure how this renders but normally when I write something like this I prefix shell commands with $ like:
```console
   $ apt-get install python-sphinx texlive-latex-recommended dvipng librsvg2-bin \
   imagemagick docbook2x graphviz
   $ python -m pip install sphinx-math-dollar
```

Reply:
I think it's better to not use the `$` prefix when showing shell commands in docs. That way people can copy paste it and it will work.

Reply:
If I just did a line break there instead would that help?